flowkit_jwt_generator/tests/test_keypairs.py

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import base64
+
+import pytest
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+
+from flowkit_jwt_generator import generate_keypair
+from flowkit_jwt_generator.jwt import load_private_key, load_public_key
+
+
+def test_keypair_generator():
+    """
+    Keypair generator produces a valid key pair, and we can load it from a string.
+    """
+    private_key, public_key = generate_keypair()
+    rsa_private_key = load_private_key(private_key.decode())
+    assert private_key == rsa_private_key.private_bytes(
```

Comment:
Isn't this simply duplicating the implementation? Still good to have the test, I just wonder whether there is an independent way of testing it (e.g. to encrypt & decrypt a small sample message and test that we get the original result?) But if not this is probably the easiest way to test things.